lib.default/misc/spell_dams

```diff
@@ -1,1601 +1,16763 @@
-Spell dam file:
```

Comment:
The changes to this file look weird.  Are they correct?

Reply:
yes?